docs/project/contributing.md

```diff
@@ -130,3 +130,20 @@ Any user-facing change should be labeled with `include-changelog`.
 The PR title should contain a concise summary of the feature or fix and the description should have the GitHub issue number.
 When we publish a new version of lakeFS, we will add this to the relevant version section of the changelog.
 If the change should not be included in the changelog, label it with `exclude-changelog`.
+
+### User-Facing Changes Examples
+1. UI/UX modifications: Changes to the layout, color scheme, or navigation structure.
+1. New features: Adding functionality that users can directly interact with.
```

Comment:
Unless defined as internal. What about experimental features?

Reply:
mm good point, what do you think, a little bit controversial

Reply:
- IMHO experimental changes are "user-visible", just marked "experimental".  This gives users a chance to let us know if they want them or not.